ODataConnectedService/src/CodeGeneration/V3CodeGenDescriptor.cs

```diff
@@ -1,16 +1,17 @@
 ﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
 // Licensed under the MIT License.  See License.txt in the project root for license information.
 
+using EnvDTE;
```

Comment:
I think you are sorting with pure alphabetical instead of "system first" which is what we do in most code. I don't think you need this change.

Reply:
Same for other files.

Reply:
Done.